test/functional/lua/iter_spec.lua

```diff
@@ -10,7 +10,7 @@ describe('vim.iter', function()
     rb:push("b")
 
     local it = vim.iter(rb)
-    eq({"a", "b"}, it:totable())
+    eq({ "a", "b" }, it:totable())
```

Comment:
Here and throughout: let's leave these kinds of formatting changes out of this PR, as it makes it more difficult to actually review what has changed. Test files are intentionally _not_ autoformatted right now, but that will (probably) change in the future, at which point these kinds of changes will be done.

Reply:
sorry about that, I wrongly assumed the tests were autoformatted and formatted before pushing. should be fixed now